cypress/integration/App.spec.ts

```diff
@@ -1,6 +1,14 @@
+/// <reference types="cypress-axe" />
+
 describe('Open aukiolot app', () => {
```

Comment:
I get the following lint warning about this in my IDE: 
`TS1208: All files must be modules when the '--isolatedModules' flag is provide`

Reply:
<img width="622" alt="Näyttökuva 2020-9-23 kello 17 23 55" src="https://user-images.githubusercontent.com/1610860/94025828-9d6e7a00-fdc1-11ea-8a56-5c8fa9ae8886.png">

Reply:
This should be fixed now